docs/_core/intro.story.tsx

```diff
@@ -10,23 +10,23 @@ export const Intro = () => (
 
     <h2>Install</h2>
     <p>
-      You can install with <code>npm</code>:
+      You can install from <code>npm</code> registry:
```

Comment:
There are many mirror registries like `yarn`'s.
I think it's better to say "install **with** the `npm` cli tool", the registry doesn't matter too much.